docs/submission/dictionary-overview.md

```diff
@@ -1,58 +1,55 @@
+Field listings can be filtered by Data Tier and Attribute, which can help identify which fields are necessary for [clinical data completion](clinical-data-validation-rules).
 
-The lists can be filtered by Data Tier and Attribute, which can help a data submitter identify which fields are necessary for [clinical data completion](clinical-data-validation-rules). 
+You can explore previous dictionary versions using the dropdown at the top of the dictionary. Using the latest version of the dictionary is required during data submission.
 
-The dictionary version appears in a dropdown above the lists, as well as a date of when it was last updated. You can explore previous dictionary versions by choosing another version number from the dropdown. This dictionary version will also be reflected in the names of the file templates that are downloaded from the Platform. It is important to use the most current files for the clinical validation to run smoothly. 
+### Field Descriptors
 
-### Field Name & Description
-Each field has a name and a description, the name being the same label that appears in the headers of each TSV file. 
+Each field has a data tier and an attribute classification, which reflects the importance of the field in terms of clinical data completion.
 
-### Data Tier & Attributes 
-Each field has a data tier and an attribute classification, which reflects the importance of the field in terms of clinical data completion. 
+![ID](/assets/submission/dictionary-id.svg) attribute indicates:
 
-#### ID Fields
-An ![ID](/assets/submission/dictionary-id.svg) field is a unique identifier that is used for cross file validation. Most ids have a ![Required](/assets/submission/dictionary-required.svg) attribute, which means they are required to be provided in an uploaded clinical file in order for a submission to be valid. 
+- An field is a unique identifier that is used for cross file validation.
+- This field is a primary or foreign key.
 
-#### Core Fields
-![Core](/assets/submission/dictionary-core.svg) clinical fields are required for each donor in order to be accepted as an ICGC ARGO case. The set of core clinical fields were defined by the [Tissue & Clinical Annotations Working Group](http://www.icgc-argo.org/page/84/tissue-clinical-annotation-working-group) which involved regular discussions with members of the working group and ARGO Programs. 
+![Conditional](/assets/submission/dictionary-conditional.svg) attribute indicates:
 
-These core clinical fields are commonly acquired in cohort-based studies and clinical trials and are required to address clinically relevant topics by cross entity analyses, and therefore constitute a critical element in the analysis of diverse ARGO Programs.
+- Field must meet certain conditions, depending on the value of another field.
```

Comment:
- `This` must meet certain conditions, depending on the value of another field.
- The conditions are described in the data dictionary notes & scripts

Reply:
rewording suggestions for sentence completion